app/src/main/java/team7/voluntime/Domains/Volunteer.java

```diff
@@ -0,0 +1,81 @@
+package team7.voluntime.Domains;
+
+/*
+
+Volunteers:
+  <VolunteerID>:
+    Profile:
+      FullName: String
+      Address: String
+      PhoneNumber: String
+      Gender: String
+      DateOfBirth: String
+      Email: String
+ */
+public class Volunteer {
+    private String id;
+    private String DateOfBirth;
+    private String FullName;
+    private String Address;
+    private String PhoneNumber;
+    private String Gender;
+    private String Email;
```

Comment:
We'll have to fix these up on Monday, I've changed the database so that the values in the database are lowercase, however, because you haven't merged my changes in the way you've got is correct for now.